src/test/resources/processExecution/recorder.ts

```diff
@@ -0,0 +1,128 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for details.
+
+import * as child_process from "child_process";
+import * as events from "events";
+import * as fs from "fs";
+import * as os from "os";
+import * as path from "path";
+
+import {ISpawnOptions} from "./recording";
```

Comment:
Sorry for the nit but could you mege this import with the other imports you already have frrom `./recording` 2 lines below? :)